docker-compose.yml

```diff
@@ -9,8 +9,9 @@ services:
       - POSTGRES_DB=${POSTGRES_DB-mathesar_django}
       - POSTGRES_USER=${POSTGRES_USER-mathesar}
       - POSTGRES_PASSWORD=${POSTGRES_PASSWORD-mathesar}
+    # db container is exposed to the other containers, and the host.
     ports:
-      - ${POSTGRES_HOST-5432}:5432
+      - "${POSTGRES_HOST-5432}:5432"
```

Comment:
Also, adds quotes around port mapping declarations, to prevent YML's base 60 bug: see the note box in the `ports` section of `docker-compose.yml` docs: https://docs.docker.com/compose/compose-file/compose-file-v3/#ports